hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/AvroDFSSource.java

```diff
@@ -45,6 +45,7 @@
   public AvroDFSSource(TypedProperties props, JavaSparkContext sparkContext, SparkSession sparkSession,
       SchemaProvider schemaProvider) throws IOException {
     super(props, sparkContext, sparkSession, schemaProvider);
+    sparkContext.hadoopConfiguration().set("avro.schema.input.key",schemaProvider.getSourceSchema().toString());
```

Comment:
space after `,` ? checkstyle is not complaining?

Reply:
Fixed it